chatbot/models.py

```
@@ -0,0 +1,506 @@
+    
+    RESPONSE_CATEGORIES = [
+        ('greeting', 'خوشامدگویی'),
+        ('symptom_inquiry', 'پرسش علائم'),
+        ('medication_info', 'اطلاعات دارو'),
+        ('appointment_booking', 'نوبت‌گیری'),
+        ('emergency', 'اورژانس'),
+        ('general_health', 'سلامت عمومی'),
+        ('farewell', 'خداحافظی'),
+        ('error', 'خطا'),
+        ('unknown', 'نامشخص'),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)